src/types/table.ts

```diff
@@ -27,7 +27,10 @@ export type TableColumnSlotName = `column-${string}`
 
 export type SwrvState = 'VALIDATING' | 'VALIDATING_HAS_DATA' | 'PENDING' | 'SUCCESS' | 'SUCCESS_HAS_DATA' | 'ERROR' | 'STALE_IF_ERROR'
 
+export type EmitState = 'loading' | 'error' | 'success'
+
 export interface SwrvStateData {
   hasData: boolean
-  state: SwrvState
+  state: SwrvState,
+  emitState: EmitState
```

Comment:
question: why is `emitState` now inside of this `SwrvStateData`? This state has nothing to do with `swrv`